components/card/PeopleCard.tsx

```diff
@@ -0,0 +1,130 @@
+"use client"
+
+import React, { useState } from "react"
+import Image from "next/image"
+import {
+  Card,
+  CardContent,
+  CardTitle,
+  CardDescription,
+} from "@/components/ui/Card"
+import {
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/Dialog"
+
+import { FaGithub, FaInfoCircle, FaTimes } from "react-icons/fa"
+import ButtonLink from "@/components/ui/ButtonLink"
+
+interface PeopleCardProps {
+  className?: string
+  imagePath: string
+  hoverImagePath?: string
+  name: string
+  title: string
+  personDetails?: {
+    name: string
+    title: string
+    team?: string
+    subteam?: string
+    type?: string
+    github_username?: string
+    brown_directory_uuid?: string
+    bio?: string
+    image?: string
+  }
+}
+
+export const PeopleCard: React.FC<PeopleCardProps> = ({
+  imagePath,
+  hoverImagePath,
+  name,
+  title,
+  personDetails,
+}) => {
+  const [isHovered, setIsHovered] = useState(false)
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
```

Comment:
There is a grey box outline for the xi n the top right corner. It shows up after I click the person on the about us page.

<img width="561" height="997" alt="Screenshot 2025-08-01 at 3 21 39 PM" src="https://github.com/user-attachments/assets/dc705dbf-29cc-47bf-8967-a1aad122dbee" />